src/main/java/io/github/thebusybiscuit/slimefun4/api/items/HashedArmorpiece.java

```diff
@@ -61,8 +61,8 @@ public void update(@Nullable ItemStack stack, @Nullable SlimefunItem item) {
             this.hash = copy.hashCode();
         }
 
-        if (item instanceof SlimefunArmorPiece) {
-            this.item = Optional.of((SlimefunArmorPiece) item);
+        if (item instanceof SlimefunArmorPiece slimefunArmorPiece) {
+            this.item = Optional.of(slimefunArmorPiece);
```

Comment:
We do not always have to be so literal about it.
```suggestion
        if (item instanceof SlimefunArmorPiece armorPiece) {
            this.item = Optional.of(armorPiece);
```